misk-redis/src/main/kotlin/misk/redis/RealRedis.kt

```diff
@@ -47,6 +47,24 @@ class RealRedis(private val jedisPool: JedisPool) : Redis {
     }
   }
 
+  // Get a map of field -> value pairs for the given key.
```

Comment:
Could these be proper kotlindoc comments?

```
/** Get a map ... */
```

Reply:
good call, updated :)